README.md

```diff
@@ -27,17 +27,15 @@ The Tye extension lets you view and debug Tye services.
 ### View running Tye application
 You can view the running Tye application, view all services and it's replicas, view logs and open the services in the browser.
 
-![View running Tye application](resources/readme/tye-view.gif)
+[![View running Tye application](https://img.youtube.com/vi/-UQau95Fz6U/0.jpg)](https://www.youtube.com/watch?v=-UQau95Fz6U)
```

Comment:
Would it make sense to use ms.aka links rather than direct links, just in case we need to redirect them elsewhere?

Reply:
That's a great suggestion!
Updated the links to the aka.ms links.